cumulus/client/service/src/lib.rs

```diff
@@ -560,11 +560,15 @@ where
 				.map_err(|e| format!("{e:?}"))?
 				.ok_or("Could not find parachain head in relay chain")?;
 
-			let target_block = B::Header::decode(&mut &validation_data.parent_head.0[..])
+			let finalized_header = B::Header::decode(&mut &validation_data.parent_head.0[..])
 				.map_err(|e| format!("Failed to decode parachain head: {e}"))?;
 
-			log::debug!(target: LOG_TARGET_SYNC, "Target block reached {:?}", target_block);
-			let _ = sender.send(target_block);
+			log::info!(
+				"🎉 Received finalized parachain header #{} ({}) from the relay chain.",
+				finalized_header.number(),
+				finalized_header.hash()
+			);
```

Comment:
The syncing logs already inform us about the progress?

Reply:
Yeah it is not an absolute necessity, you already get the occasional `⏩ Waiting for pending target block (7 peers), best: #0 (0x4823…771a), finalized #0 (0x4823…771a)`. However this here is tells you which block is the new target and is immediately printed when we retrieve the finalized one. I am also against spamming logs, but in my opinion this additional info does not hurt.

Reply:
But then maybe change the text to `Received target parachain header ..` or something like this.